docs/bots-guide.md

```diff
@@ -99,15 +99,15 @@ You need:
     * To subscribe your bot to streams, navigate to *Manage
       Streams*. Select a stream and add your bot by its email address
       (the address you assigned in step 3).
-    * Now, the bot do its job on the streams you subscribed it to.
+    * Now, the bot can do its job on the streams you subscribed it to.
     * (In future versions of the API, this step may not be required).
 
 4. Run the bot.
 
     * In your Zulip repository, navigate to `~/zulip/contrib_bots/`
     * Run
       ```
-      python run.py ~/zulip/contrib_bots/bots/<my-bot>/<my-bot>.py --config-file ~/.zuliprc`
+      python run.py bots/<my-bot>/<my-bot>.py --config-file ~/.zuliprc`
```

Comment:
We would also want to change `.zuliprc` to `zuliprc`, since that's what's being downloaded from your zulip server.